ResidentInformationApi/V1/Controllers/ResidentInformationController.cs

```diff
@@ -0,0 +1,18 @@
+using Microsoft.AspNetCore.Mvc;
+using ResidentInformationApi.V1.Boundary.Requests;
+
+namespace ResidentInformationApi.V1.Controllers
+{
+    [ApiController]
+    [ApiVersion("1.0")]
+    [Route("api/v1/residents")]
+    [Produces("application/json")]
+    public class ResidentInformationController : BaseController
+    {
+        [HttpGet]
+        public IActionResult ListContacts([FromQuery] ResidentQueryParam rqp)
+        {
+            return Ok(rqp.ToString());
```

Comment:
How come we are calling `.ToString()` here? I think this will just result to the object type name, not it's contents